dj_cqrs/controller/consumer.py

```diff
@@ -32,7 +33,7 @@ def route_signal_to_replica_model(signal_type, cqrs_id, instance_data, previous_
     :param dict instance_data: Master model data.
     """
     if signal_type not in (SignalType.DELETE, SignalType.SAVE, SignalType.SYNC):
-        logger.error('Bad signal type "{}" for CQRS_ID "{}".'.format(signal_type, cqrs_id))
+        logger.error('Bad signal type "{0}" for CQRS_ID "{1}".'.format(signal_type, cqrs_id))
```

Comment:
Required index parameters don't make code clear and doest help (we still get F524 on missing parameters).
Maybe it's better ignore P101?

Reply:
I consider them helpful, so will leave as is.